cert-manager-configs/.openshift/route-with-cert.yml

```diff
@@ -0,0 +1,56 @@
+---
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  annotations:
+    description: Vanity Certificate and Configuration to manage a vanity route with a valid cert
+  name: "vanity-route-certificate"
+labels:
+  template: "vanity-route-certificate"
+objects:
+  - kind: Certificate
+    apiVersion: cert-manager.io/v1alpha2
+    metadata:
+      name: "${NAME}-certificate"
+      namespace: ${NAMESPACE}
+    spec:
+      secretName: "${NAME}-certificate"
+      issuerRef:
+        name: letsencrypt-production
+        kind: ClusterIssuer
```

Comment:
Can we make these parameters, similar to the [Helm template](https://github.com/redhat-cop/containers-quickstarts/blob/master/cert-manager-configs/templates/Certificate.yaml#L12-L13)?

Reply:
@pabrahamsson done